model/validation.go

```diff
@@ -49,7 +49,8 @@ func IsDNS1123Label(value string) bool {
 	return len(value) <= dns1123LabelMaxLength && dns1123LabelRex.MatchString(value)
 }
 
-func validatePort(port int) error {
+// ValidatePort ...
+func ValidatePort(port int) error {
```

Comment:
Might be good to set precedent here and start with proper comments, otherwise it's more likely external contributions may copy existing incorrect style.